README.md

```diff
@@ -85,6 +85,7 @@ MODELS=`[
     "datasetName": "OpenAssistant/oasst1",
     "description": "A good alternative to ChatGPT",
     "websiteUrl": "https://open-assistant.io",
+    "endpoints": [{"url": "http://127.0.0.1:8080/generate_stream"}],
```

Comment:
So it would be dope if we could leave it out of the default `.env` and then add a note under "Running your own models" in the readme about the `endpoints` key (with the example you gave)

Reply:
Good point, I will make the requested changes

Reply:
I have removed the requested information and updated the readme. Let me know if you have any other concerns or feedback.